lib/ansible/modules/commands/command.py

```diff
@@ -123,12 +123,14 @@
     creates: /path/to/database
 
 # 'argv' is a parameter, indented one level from the module
+# When using this syntax, you don't need the "args" block anymore
```

Comment:
I think this comment can be improved.  It should indicate that when not using "freeform" arguments, such as the following, that `args` is not necessary:

```
- command: /usr/bin/make_database.sh
```